src/main/java/com/minecolonies/coremod/inventory/InventoryCitizen.java

```diff
@@ -418,11 +423,6 @@ public String getName()
         return this.hasCustomName() ? this.customName : "citizen.inventory";
     }
 
-    public MaterialStore getMaterialStore()
-    {
-        return materialStore;
-    }
-
     private void addStackToMaterialStore(@Nullable final ItemStack stack)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "addStackToMaterialStore" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUnusedPrivateMethod)